duckduckgo/duckduckgo-privacy-extension
# Increase placeholder threshold to 30

**Reviewer:** @laurengarcia 



## Description:
Don't show top blocked stats until at least 30 pages have been visited (increased from 10)


## Steps to test this PR:
1. Build and either reload and reset stats, or remove existing and load again
2. Navigate to some sites. After 10 page loads, the main popup should still show the placeholder
3. Verify top blocked appear after 30 page loads

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
